docs/source/routing/performance/caching/response-caching/quickstart.mdx

```diff
@@ -0,0 +1,267 @@
+    import: ["@key", "@external"]
+  )
+
+type Query {
+  user(id: ID!): User
+  users: [User!]!
+}
+
+type User @key(fields: "id") {
+  id: ID!
+  name: String!
+  email: String!
+  posts: [Post!]! @external
+}
+
+type Post @key(fields: "id") {
+  id: ID!
+  content: String! @external
+}
+```
+
+```graphql title="posts.graphql"
+extend schema
+  @link(url: "https://specs.apollo.dev/federation/v2.12", import: ["@key"])
+
+type Query {
+  posts: [Post!]
+  post(id: ID!): Post
+}
+
+type User @key(fields: "id") {
+  id: ID!
+  posts: [Post!]!
+}
+
+type Post @key(fields: "id") {
+  id: ID!
+  title: String!
+  content: String!
+  author: User!
+  featuredImage: String
+}
+```
+
+As I saw earlier with metrics I could highly benefit from caching by caching type `User` on subgraph `posts` I'm going to do it by using the `@cacheControl` directive.
+
+<Note>
+    This example using `@cacheControl` only works if you're using Apollo Server as subgraph. If you're using something else then you'll have to update your codebase to return the right `Cache-control` header and it will work properly.
+</Note>
+
+Here is the new version of the schema for subgraph `posts`:
+
+```graphql title="posts.graphql"
+extend schema
+  @link(url: "https://specs.apollo.dev/federation/v2.12", import: ["@key"])
+
+enum CacheControlScope {
+  PUBLIC
+  PRIVATE
+}
+directive @cacheControl(
+  maxAge: Int
+  scope: CacheControlScope
+  inheritMaxAge: Boolean
+) on FIELD_DEFINITION | OBJECT | INTERFACE | UNION
+
+type Query {
+  posts: [Post!] @cacheControl(maxAge: 60)
+  post(id: ID!): Post
+}
+
+type User @key(fields: "id") @cacheControl(maxAge: 60) {
+  id: ID!
+  posts: [Post!]!
+}
+
+type Post @key(fields: "id") @cacheControl(maxAge: 60) {
+  id: ID!
+  title: String!
+  content: String!
+  author: User!
+  featuredImage: String
+}
+```
+
+I set the cache control with TTL of 60 seconds so both the `Post` and `User` types will be in cached for 60 seconds.
+Now I enabled this you can see the difference on the previous dashboard, you now have cache hits and less cache misses.
+
+<img
+  className="screenshot"
+  alt="Response cache hit metrics dashboard"
+  src="../../../../images/response-cache/cache_hit_dashboard.png"
+/>
+
+And the response time will be faster as you can see on the right in the following screenshot:
+
+<img
+  className="screenshot"
+  alt="Response cache metrics dashboard"
+  src="../../../../images/response-cache/cache_resp_time.png"
+/>
+
+### Invalidation
+
+Now you have cached data based on TTL maybe you would like to be able to increase these TTLs and automatically invalidate the data you want when you know there's a change on a specific data. By default we have several ways to invalidate data and to go further you can read the docs about [invalidation](./invalidation) but for this quickstart we will use cache tags which are like surrogate cache keys for REST APIs. You can tag some data on your schema to tag it and be able to only invalidate this data.
```

Comment:
This link doesn't work for some reason. Navigating to "Invalidation" from the nav bar also does't work.

Reply:
this should work now.